From 4b539ce398196297c6e0c9503e0a339ef5605962 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:16 -0400
Subject: [fs] cifs: Fix unused variable warning when CIFS POSIX disabled

Message-id: <1406549822-10376-24-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86787
O-Subject: [RHEL7.1 PATCH 23/69] [bz1082049] Fix unused variable warning when CIFS POSIX disabled
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Steve French <smfrench@gmail.com>

Fix unused variable warning when CONFIG_CIFS_POSIX disabled.

   fs/cifs/ioctl.c: In function 'cifs_ioctl':
>> fs/cifs/ioctl.c:40:8: warning: unused variable 'ExtAttrMask' [-Wunused-variable]
     __u64 ExtAttrMask = 0;
           ^
Pointed out by 0-DAY kernel build testing backend

Signed-off-by: Steve French <smfrench@gmail.com>
(cherry picked from commit f10d9ba405ed250d647d6d810f8fc5ff2f62f370)

Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/ioctl.c b/fs/cifs/ioctl.c
index 0298670..ba54bf6 100644
--- a/fs/cifs/ioctl.c
+++ b/fs/cifs/ioctl.c
@@ -37,7 +37,6 @@ long cifs_ioctl(struct file *filep, unsigned int command, unsigned long arg)
  struct cifsFileInfo *pSMBFile = filep->private_data;
  struct cifs_tcon *tcon;
  __u64 ExtAttrBits = 0;
- __u64 ExtAttrMask = 0;
  __u64   caps;
 
  xid = get_xid();
@@ -54,6 +53,7 @@ long cifs_ioctl(struct file *filep, unsigned int command, unsigned long arg)
    caps = le64_to_cpu(tcon->fsUnixInfo.Capability);
 #ifdef CONFIG_CIFS_POSIX
    if (CIFS_UNIX_EXTATTR_CAP & caps) {
+    __u64 ExtAttrMask = 0;
     rc = CIFSGetExtAttr(xid, tcon,
           pSMBFile->fid.netfid,
           &ExtAttrBits, &ExtAttrMask);
-- 
1.7.1